frontend/online-shop/src/theme.tsx

```diff
@@ -0,0 +1,30 @@
+import { createMuiTheme } from "@material-ui/core/styles";
+
+const googleFont = "'Quicksand', sans-serif";
```

Comment:
You could place those directly within the theme as you don't use them anywhere else